src_c/math.c

```diff
@@ -3858,6 +3891,7 @@ vector_elementwiseproxy_pow(PyObject *baseObj, PyObject *expoObj,
     if (vector_elementwiseproxy_Check(baseObj)) {
         dim = ((vector_elementwiseproxy *)baseObj)->vec->dim;
         tmp = ((vector_elementwiseproxy *)baseObj)->vec->coords;
+        vec = ((vector_elementwiseproxy *)baseObj)->vec;
```

Comment:
```suggestion
        vec = ((vector_elementwiseproxy *)baseObj)->vec;
        dim = vec->dim;
        tmp = vec->coords;
```
For better code clarity